sheets/systemd-sysvinit

```diff
@@ -71,7 +71,7 @@ systemctl daemon-reload
 # SystemD Target:           runlevel1.target
 #                           rescue.target
 #
-# User-defined/Site-specific runlevels. By default, identical to 3.
+# User-defined/Site-specific run levels. By default, identical to 3.
```

Comment:
This is likely shown as "runlevels" because that's how it's displayed throughout documentation and the command's man page itself is written as such: runlevel(8) Granted, it having a space wouldn't be valid. Lol When it comes to computer-speak, there can be an important difference between correct English and the correct terms, IMO.

Reply:
I believe, *runlevel* is the only correction that I would revert here. In my opinition *runlevel* is correct and *run level* is wrong (contrary to *mount point*, *command line*, etc. where the fixes  make sense). See, for example:

https://en.wikipedia.org/wiki/Runlevel
https://www.wordsense.eu/runlevel/